README.md

```diff
@@ -6,82 +6,82 @@ OpenShift Application Platform
 
 ### Origin API
 
-The Origin API is located on each server at `https://<host>:8443/oapi/v1`. These APIs are described via [Swagger v1.2](https://www.swagger.io) at `https://<host>:8443/swaggerapi/oapi/v1`. For more, [see the API documentation](http://docs.openshift.org/latest/rest_api/openshift_v1.html).
+The Origin API is located on each server at `https://<host>:8443/oapi/v1`. These APIs are described via [Swagger v1.2](https://www.swagger.io) at `https://<host>:8443/swaggerapi/oapi/v1`. For more, [see the API documentation](https://docs.openshift.org/latest/rest_api/openshift_v1.html).
 
 ### Kubernetes
 
-If you're looking for more information about using Kubernetes or the lower level concepts that Origin depends on, see the following:
+Since OpenShift Origin builds on Kubernetes, it is helpful to understand underlying concepts such as Pods and Replication Controllers. The following are good references:
 
-* [Kubernetes Getting Started](https://github.com/kubernetes/kubernetes/blob/master/README.md)
+* [Kubernetes User Guide](http://kubernetes.io/docs/user-guide/)
+* [Kubernetes Getting Started](http://kubernetes.io/docs/whatisk8s/)
 * [Kubernetes Documentation](https://github.com/kubernetes/kubernetes/blob/master/docs/README.md)
-* [Kubernetes API](http://docs.openshift.org/latest/rest_api/kubernetes_v1.html)
+* [Kubernetes API](https://docs.openshift.org/latest/rest_api/kubernetes_v1.html)
 
 ### Troubleshooting
 
-If you run into difficulties running Origin, start by reading through the [troubleshooting guide](https://github.com/openshift/origin/blob/master/docs/debugging-openshift.md).
-
+The [troubleshooting guide](https://github.com/openshift/origin/blob/master/docs/debugging-openshift.md) provides advice for diagnosing and correcting any problems that you may encounter while installing, configuring, or running Origin.
 
 FAQ
 ---
 
 1. How does Origin relate to Kubernetes?
 
-    Origin is a distribution of Kubernetes optimized for enterprise application development and deployment,
-    used by OpenShift 3 and Atomic Enterprise.  Origin embeds Kubernetes and adds additional
-    functionality to offer a simple, powerful, and easy-to-approach developer and operator experience
-    for building applications in containers.  Our goal is to do most of that work upstream, with
-    integration and final packaging occurring in Origin.
+    Origin is a distribution of Kubernetes optimized for enterprise application development and deployment, and is the foundation of OpenShift 3 and Atomic Enterprise.  Origin extends Kubernetes with additional functionality, offering a simple, yet powerful, development and operational experience.  Both Origin and the upstream Kubernetes project focus on building and deploying applications in containers.
```

Comment:
Remove '3 and Atomic Enterprise'.

Kubernetes doesn't have builds.

Not sure the overall goal of changing the last sentence.  It has a very different meaning.